policyengine_us/parameters/gov/states/ar/tax/income/deductions/itemized/tuition/weighted_average_tuition/two_year_college.yaml

```diff
@@ -2,7 +2,7 @@ description: Arkansas defines the weighted average tuition for four-year college
 values:
   2021-01-01: 2_141
   2022-01-01: 2_272
-
+  2023-01-01: 4_405
```

Comment:
shouldn't this be [this](https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/AR1075_2023.pdf#page=1) number:

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/a8d3aad6-b344-4afa-934c-daab0df1af07)